include/swift/Markup/AST.h

```diff
@@ -589,6 +589,32 @@ class Image final : public InlineContent,
   }
 };
 
+class Attribute final : public InlineContent, private llvm::TrailingObjects<Image, MarkupASTNode *> {
```

Comment:
Since there are a lot of things called `Attribute` in this codebase, can we be a little more descriptive, like `AttributedInlineContainer`?

Reply:
Also, I think it would make it a little clearer to describe what is the attribute and what contains them. This type makes it seem like attributes are represented structurally.

Reply:
I went with `InlineAttributes`, to mirror swift-cmark.

Reply:
👍